From 8fa7f2e063b675f7e1236af329dbb911bffe95d6 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Tue, 19 Nov 2024 10:18:37 +0200
Subject: [PATCH] [FSW-11350] fs1: remove unused sdk cflags

With this patch removed compiler flags those
are not related to bootrom, and used in previous
projects for sdk related modules build.

Change-Id: Ice2cd6e675e61e557a7f4419155ac8ef3226be05
---

diff --git a/Makefile b/Makefile
index 63f8ff5..b25c8e6 100644
--- a/Makefile
+++ b/Makefile
@@ -62,9 +62,6 @@
 LDFLAGS_IO = $(LDFLAGS)
 LDFLAGS_IO += -T $(HL_DEV_ARCH)/$(HL_TARGET)_$@.lds
 
-CFLAGS = -DELP_ARCV2
-export CFLAGS += -Os -fdata-sections -ffunction-sections
-
 BOOT_ROM_COMMON_SRCS_C := $(shell ls common/*.c common/rsa/*.c)
 BOOT_ROM_COMMON_SRCS_C += $(shell ls $(HL_DEV_ARCH)/common.c)
 BOOT_ROM_COMMON_SRCS_C += $(shell ls $(HL_DEV_ARCH)/$(HL_TARGET).c)
